### DISCUSSION TITLE
Welcome to MathPHP Discussions!

### DISCUSSION THREAD

**Discussion by markrogoyski** [markrogoyski/math-php#438] [Announcements]
I'm opening up the Github discussions to move and start new discussions here rather than in the Issues tab. I'd like the issues to be for bugs and concrete feature requests. Everything else can become a discussion.

**Comment by pascualstromsnes** [comment]
I stumbled upon your work while pondering about the status of PHP as a language for math and science. 
I am really impressed with the work done here. Thank you ever so much for your time and dedication!
